dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue

```diff
@@ -34,6 +34,9 @@
             </template>
           </Column>
           <Column field="companyInformation.headquarters" header="LOCATION" :sortable="true" class="surface-0 w-2">
+            <template #body="{data}">
+              {{ buildLocationString(data.companyInformation.headquarters, data.companyInformation.countryCode) }}
```

Comment:
Im Vue-Kurs wurde gesagt:
Everything (especially methods) in the HTML Code that is not linked to a specific event is re-executed by Vue each time any event occurs, because Vue does not know which methods uses which variable, and variables might be updated in the event, so it re-executes all those methods.

Vllt können wir hier eine Computed Property bauen?

Reply:
Aus Prio-Gründen so belassen